go.mod

```diff
@@ -3,49 +3,53 @@ module github.com/k8ssandra/medusa-operator
 go 1.15
 
 require (
-	github.com/bombsimon/logrusr v1.1.0 // indirect
-	github.com/go-logr/logr v0.1.0
-	github.com/golang/protobuf v1.4.3 // indirect
+	github.com/bombsimon/logrusr v1.1.0
+	github.com/go-logr/logr v0.4.0
 	github.com/google/uuid v1.1.2
-	github.com/k8ssandra/cass-operator v1.7.0
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
-	github.com/sirupsen/logrus v1.8.1 // indirect
-	github.com/stretchr/testify v1.7.0 // indirect
-	golang.org/x/net v0.0.0-20201209123823-ac852fbbde11 // indirect
-	golang.org/x/sys v0.0.0-20201214210602-f9fddec55a1e // indirect
-	golang.org/x/text v0.3.4 // indirect
-	google.golang.org/genproto v0.0.0-20201214200347-8c77b98c765d // indirect
-	google.golang.org/grpc v1.34.0
-	google.golang.org/protobuf v1.25.0
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
+	github.com/k8ssandra/cass-operator v1.8.0
+	github.com/sirupsen/logrus v1.8.1
+	github.com/stretchr/testify v1.7.0
+	google.golang.org/grpc v1.38.0
+	google.golang.org/protobuf v1.26.0
+	k8s.io/api v0.22.2
```

Comment:
This does not match 1.21.4

Reply:
Or is this version coming from something else which is why replace is only used?

Reply:
Right, I must have imported some deps I shouldn't have. I'll fix this.

Reply:
Fixed